pkg/backend/display/diff_test.go

```diff
@@ -137,3 +137,63 @@ func TestDiffEvents(t *testing.T) {
 		})
 	}
 }
+
+func TestHasPolicyViolations(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name                   string
+		events                 []engine.PolicyViolationEventPayload
+		hasMandatoryViolations bool
```

Comment:
optional: "mandatory violation" is already in context with the test name.
could rename this to just "want" (and optionally "events" to "give") to indicate input/output of the function.